translate/storage/properties.py

```diff
@@ -135,8 +135,8 @@
 from translate.misc.multistring import multistring
 from translate.storage import base
 from copy import deepcopy
-from xml.etree import ElementTree
-from xml.sax.saxutils import escape, unescape
+from lxml import etree
+from xml.sax.saxutils import unescape
```

Comment:
Is the unescape really needed? It is used in several places, but I see single test for XML entity.

Reply:
Good catch, it's actually a mistake the way I use it here. Improving the test and fixing the code.